doc/Alerting/Rules.md

```diff
@@ -28,39 +28,44 @@ __Conditions__ can be any of:
 - Smaller or Equal `<=`
 
 __Values__ can be an entity or any single-quoted data.
-__Glues__ can be either `&&` for `AND`.  or `||` for `OR`. Note if you need to use `OR` `||` please use a macro. 
+__Glues__ can be either `&&` for `AND`.  or `||` for `OR`. 
```

Comment:
Actually, we've got rid of `||` so remove it from this line.

Reply:
I know the are gone from the web ui but not from the cli?

Reply:
CLI?

Reply:
Using macros in config.php

Reply:
That's covered in the note below.